Reference/Searching/Examine/transforming-index-values.md

```diff
@@ -0,0 +1,120 @@
+---
+versionFrom: 8.0.0
+---
+
+# TransformingIndexValues
+
+In the [Quick Start](Quick-Start/index.md) documentation you can see how to perform a simple search with Examine. That is great if you want to search between node names or you know that you always want to search for a specific field - e.g. `bodyText`. 
+
+However, what if you want to search through several different node types and search across many different fields, you likely will not want to have a query that looks like this:
```

Comment:
this is assuming what the audience thinks. Instead it can just be written:

> However, what if you want to search through several different node types and search across many different fields, you will typically need to have a query that looks like this: